frontend/components/Tag/_styles.scss

```diff
@@ -0,0 +1,27 @@
+.tag {
+  display: flex;
+  height: 18px;
```

Comment:
I think we should use `px-to-rem` even when not setting as an official variable, for the same
accessiblity reasons we use it when we are setting one. Added to frontend sync agenda to discuss.